src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add: function (num) {
+      this.result += num;
+    },
```

Comment:
Instead of using the `function` keyword, use the method shorthand syntax for defining object methods.